Add JSON-LD Context Document for Data Package (#874)
frictionlessdata/specs: schemas/context/README.md

This directory contains non-normative JSON-LD Context Documents to convert Data Package Formats to RDF. 

> By now it only contains a context for Data Package documents.
> Properties `resources`, `profile`, and contributor roles are ignored.
> 
> Note that the use of POSIX pathes will reduce the ability for RDF conversion. In particular:
> 
> - property `id`, if given, MUST be an URI

Reviewer: ```suggestion
Right now, it only contains a context for Data Package documents.
Properties `resources`, `profile`, and contributor roles are ignored.

Note that the use of POSIX paths will reduce the ability for RDF conversion. In particular:

- property `id`, if given, MUST be a URI
```
